index.js

```diff
@@ -38,7 +38,9 @@ module.exports = {
 
   treeForVendor: function(tree){
     // Package up the highlight.js source from its node module.
-    var src = this.treeGenerator(path.join(__dirname, 'node_modules', 'highlight.js'));
+
+    var src = this.treeGenerator(path.join(require.resolve('highlight.js'), '..', '..'));
```

Comment:
resolves to `node_modules/highlight.js/lib/index.js` but we want `node_modules/highlight.js` this will also work with npm 2